### PR TITLE
improvement: ZENKO-4381 upgrade mongodb to fix CVE

### PR DESCRIPTION
**Pull request by naren-rajendran**
Fixes, 
CVE-2017-12424 
CVE-2017-14062
CVE-2021-3520

### CODE REVIEW

**Review by tmacro — APPROVED** [review]

File: .github/scripts/end2end/configs/mongodb_options.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ image:
   ## Bitnami MongoDB image tag
   ## ref: https://hub.docker.com/r/bitnami/mongodb/tags/
   ##
-  tag: 4.0.14-debian-9-r24
+  tag: 4.0.27-debian-9-r118
```
Comment (francoisferrand) [inline .github/scripts/end2end/configs/mongodb_options.yaml:21]: I believe changing only deps.yaml is enough, right?

Also, please remember this fixes the CVE only in non-sharded mongo, which we mostly do not use (we deploy sharded mongo)

**Review by welansari — APPROVED** [review]

File: .github/scripts/end2end/configs/mongodb_options.yaml (hunk 1 above)
Comment (naren-rajendran) [reply .github/scripts/end2end/configs/mongodb_options.yaml:21]: @francoisferrand There is another pr handling the sharded mongo bump, https://github.com/scality/Zenko/pull/1745

Comment (naren-rajendran) [reply .github/scripts/end2end/configs/mongodb_options.yaml:21]: And I am not sure if changing only deps.yaml is sufficient. So I made changes in all places non-sharded mongodb version is referred.

**Review by nicolas2bert — APPROVED** [review]

**Comment by naren-rajendran** [comment]
@bert-e ping

**Pull request merged**